pose-detection/src/calculators/low_pass_filter.ts

```diff
@@ -31,10 +31,22 @@ export class LowPassFilter {
 
   constructor(private alpha: number) {}
 
-  apply(value: number): number {
+  apply(value: number, threshold?: number): number {
     let result;
     if (this.initialized) {
-      result = this.alpha * value + (1 - this.alpha) * this.storedValue;
+      if (threshold == null) {
+        // Regular lowpass filter.
+        // result = this.alpha * value + (1 - this.alpha) * this.storedValue;
+        // We need to reformat the formula to be able to conveniently apply
```

Comment:
Maybe add that second part of the comment to the other branch, in the threshold != null case would be a bit clearer?